staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -822,7 +832,8 @@ func UpdateResource(r rest.Updater, scope RequestScope, typer runtime.ObjectType
 		if wasCreated {
 			status = http.StatusCreated
 		}
-		responsewriters.WriteObject(status, scope.Kind.GroupVersion(), scope.Serializer, result, w, req.Request)
+
+		transformResponseObject(ctx, scope, req, res, status, result)
```

Comment:
It contains logic that implements the rest protocol.  It calls into storage as needed.